WinSonic/Controls/ConfirmationContentDialog.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+
+namespace WinSonic.Controls
+{
+    public class ConfirmationContentDialog
+    {
+        public static ContentDialog CreateDialog(XamlRoot xamlRoot)
+        {
+            ContentDialog dialog = new()
+            {
+                Title = "Are you sure?",
+                Content = "This action cannot be undone.",
+                PrimaryButtonText = "Delete",
```

Comment:
Based on this text, the `DeleteConfirmationContentDialog` name would be more descriptive.